edxval/tests/test_api.py

```diff
@@ -38,7 +38,7 @@ def check_direction(sort_direction, expected_ids):
             """Assert that the given videos match the expected ids"""
 
             # Make sure it's not just returning the order given
-            actual_videos = api_func(list(reversed(expected_ids)), sort_field, sort_direction)
+            actual_videos = api_func(list(reversed(expected_ids)), sort_field, sort_direction)["videos"]
```

Comment:
I'll leave these tests as it and add optional pagination parameter to apply paging and test it through new test cases.